paper/paper.md

```diff
@@ -61,55 +77,89 @@ nf-core community [@ewels_nf-core_2020].
-storing and processing metagenomic data that new tools can plug into.
+storing and processing metagenomic data that new tools can plug into, however
+this suite of software is aimed mostly at 16S amplicon data [@cite], and thus
+often does have as much support or awareness in 'shotgun' sequencing [@cite]
+metagenomic studies . While some of the taxonomic profilers also come with
+scripts to convert their output into another format and/or merge multiple
+profiles into a single table, such as the Krakentools companion
+package[@lu_metagenome_2022], these are often focused on the specific tool or
+family of tools.
+
+Reading a wide range of formats of primarily shotgun-metagenomic profiling tools
+and formats is supported by TAXPASTA, and it is designed to be used as a
+building block in metagenomic analysis workflows. At the time of writing, it is
+able to read profiles from nine different profilers, namely Bracken
+[@lu_bracken_2017], Centrifuge [@kim_centrifuge_2016], DIAMOND
+[@buchfink_sensitive_2021], Kaiju [@menzel_fast_2016], Kraken2
+[@wood_improved_2019], KrakenUniq [@breitwieser_krakenuniq_2018], MALT/MEGAN6
+[@huson_megan_2016; @vagene_salmonella_2018], MetaPhlAn
+[@blanco-miguez_extending_2023],and mOTUs
+[@ruscheweyh_cultivation-independent_2022]. Supporting more profilers is already
+planned, and detailed documentation for community contributions is
+provided[^10].
+
+[^10]: <https://taxpasta.readthedocs.io/en/latest/contributing/supporting_new_profiler/>
+
+For maximum compatibility, TAXPASTA offers a wide range of familiar output file
+formats, such as text-based, tabular formats (CSV[^1], TSV[^2]), spreadsheets
+(ODS[^3], XLSX[^4]), optimized binary formats (Apache Arrow[^5] and
+Parquet[^6]), and the HDF5-based[^7] BIOM format [@mcdonald_biological_2012].
+We hope that this will let researchers plug and play TAXPASTA into their
+existing analysis workflows in a wide range of settings.
 
-Some of the taxonomic profilers also come with scripts to convert their output
-into another format but none of them support such a wide range of tools as
-taxpasta does.
+[^1]: <https://en.wikipedia.org/wiki/Comma-separated_values>
+[^2]: <https://en.wikipedia.org/wiki/Tab-separated_values>
+[^3]: <https://en.wikipedia.org/wiki/OpenDocument>
+[^4]: <https://en.wikipedia.org/wiki/Office_Open_XML>
+[^5]: <https://arrow.apache.org/>
+[^6]: <https://parquet.apache.org/>
+[^7]: <https://www.hdfgroup.org/solutions/hdf5/>
 
 # Acknowledgements
 
-SS was supported by Rapid establishment of comprehensive laboratory pandemic
-preparedness – RAPID-SEQ (Awarded to Prof. Jan Albert). JAFY received funding from the Werner
-Siemens-Stiftung ("Paleobiotechnology", Awarded to Prof. Pierre Stallforth and
+SS was supported by "Rapid establishment of comprehensive laboratory pandemic
+preparedness – RAPID-SEQ" (awarded to Prof. Jan Albert). MB and JAFY were
+supported by the Max Planck Society. JAFY received funding from the Werner
```

Comment:
Need to check if @maxibor is also supported by the WSS Palaeobiotech project and/or the microverse EXC

Reply:
I made a suggestion below